pasta/base/annotate.py

```diff
@@ -1222,15 +1222,14 @@ def indented(self, node, children_attr):
     indent_token = self.tokens.peek_conditional(
         lambda t: t.type == token_generator.TOKENS.INDENT)
     new_indent = indent_token.src
-    if (not new_indent.startswith(prev_indent) or
-        len(new_indent) <= len(prev_indent)):
-      raise AnnotationError(
-          'Indent detection failed (line %d); inner indentation level is not '
-          'more than the outer indentation.' % cur_loc[0])
+    new_diff = _get_indent_diff(prev_indent, new_indent)
+    if not new_diff:
```

Comment:
Just noticed that this should be setting `new_diff` to a reasonable default (four spaces). Will add.